.pre-commit-config.yaml

```diff
@@ -16,7 +16,7 @@
   - id: trailing-whitespace
 
 - repo: https://github.com/ambv/black
-  rev: stable
```

Comment:
weird, this works for me, check your pre-commit version

Reply:
ah. had older installed:

```
pre-commit 1.5.0
```